lib/mods_display/fields/imprint.rb

```diff
@@ -277,7 +277,10 @@ def process_bc_ad_dates(date_fields)
           year = date_field.text.strip.gsub(/^-0*/, '').to_i + 1
           date_field.content = "#{year} B.C."
         when date_is_ad?(date_field)
-          date_field.content = "#{date_field.text.strip.gsub(/^0*/, '')} A.D."
+          # special case: year zero is 1 A.D. see:
+          # https://github.com/sul-dlss/mods_display/issues/39#issuecomment-1012606117
+          year = [date_field.text.strip.gsub(/^0*/, '').to_i, 1].max
```

Comment:
I wonder if a separate, explicit case for '0' would be easier to grok?

Reply:
I had that thought too; happy to change it. Would the best way be:
```ruby
date_field.content = "1 A.D." if year == 0 else "#{year} A.D."
```

Reply:
I'd pull it up to another `case`, maybe e.g. :
```
when date_is_bc_edtf?(date_field)
   ...
when date_is_ad?(date_field)
   ...
when date_field.text.strip == '0'
  '1 A.D.'
```